src/microbe_stage/Microbe.Contact.cs

```diff
@@ -804,6 +804,11 @@ private IEnumerable<FloatingChunk> OnKilled()
             }
         }
 
+        // Give bonus glucose at third the amount of our compounds bag capacity
+        compoundsToRelease.TryGetValue(glucose, out float existingGlucose);
+        compoundsToRelease[glucose] = existingGlucose + Compounds.Capacity /
+            Constants.ADDITIONAL_DIGESTIBLE_GLUCOSE_AMOUNT_DENOMINATOR;
```

Comment:
As this logic is a bit complicated and used in 2 places this should be extracted as a method.